web/packages/teleport/src/UnifiedResources/ResourceCard.tsx

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { Box, ButtonBorder, Flex, Label, Text } from 'design';
+
+import { CheckboxInput } from 'design/Checkbox';
+import { ResourceIcon, ResourceIconName } from 'design/ResourceIcon';
+import {
+  ApplicationsIcon,
+  DatabasesIcon,
+  DesktopsIcon,
+  KubernetesIcon,
+  ServersIcon,
+} from 'design/SVGIcon';
+
+import { UnifiedResource, UnifiedResourceKind } from 'teleport/services/agents';
+
+const SingleLineBox = styled(Box)`
+  overflow: hidden;
+  white-space: nowrap;
+`;
+
+const TruncatingLabel = styled(Label)`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`;
+
+type Props = {
+  resource: UnifiedResource;
+};
+export const ResourceCard = ({ resource }: Props) => {
+  const name = resourceName(resource);
+  const resIcon = resourceIconName(resource);
+  const ResTypeIcon = resouceTypeIcon(resource.kind);
+  const description = resourceDescription(resource);
+  return (
+    <CardContainer p={3} alignItems="start">
+      <CheckboxInput type="checkbox" mx={0}></CheckboxInput>
+      <ResourceIcon
+        alignSelf="center"
+        name={resIcon}
+        width="60px"
+        height="60px"
```

Comment:
you probably followed the figma design, but i think smaller icon works better, we can fit a few extra letters 😅 and looks less "in your face"

what do you think of `45px` instead?

Reply:
45px works fine, changed.